src/pages/_app.tsx

```diff
@@ -1,18 +1,46 @@
+import { useEffect } from 'react';
+import Script from 'next/script';
+import { useRouter } from 'next/router';
 import type { AppProps } from 'next/app';
 import Head from 'next/head';
 import theme from '@mdb/flora/theme';
 import { ThemeProvider } from '@theme-ui/core';
-
+import { GTM_ID, pageView } from '../lib/gtm';
 import Layout from '../components/layout';
 
 function MyApp({ Component, pageProps }: AppProps) {
+    {
+        /* Is used to track route changes and send page views to Google Tag Manager */
+    }
```

Comment:
@gfunk1230 I approved but wanted to point out that the brackets aren't needed here.

Reply:
Good catch! will remove!